core/digital-twin-lib/src/DAppContract.ts

```diff
@@ -112,13 +112,17 @@ export class DAppContract {
    */
   async loadBaseInfo(): Promise<void> {
     await (this as any).ensureContract();
+    const [contractAddress, description, ownerAddress] = await Promise.all([
+      (this as any).getContractAddress(),
```

Comment:
shouldn't it work actually without any, we are extending the Container class...

Reply:
Nope. It's the base class for the DAppTwin and the DAppContainer. At this point, the class does not know, that this function exists.